typings/config.d.ts

```diff
@@ -7,29 +7,30 @@
 import Gatherer = require('../lighthouse-core/gather/gatherers/gatherer.js');
 import Audit = require('../lighthouse-core/audits/audit.js');
 
-
 declare global {
   module LH {
     /**
      * The full, normalized Lighthouse Config.
      */
-    export interface Config {
+    export interface Config extends Config.Json {
```

Comment:
this will give the very nice property that a `Config` is a valid `Config.Json`. Will allow easy serialization, `--print-config` or whatever, and means you don't have to keep track of what you have, a real config or a configJson. Either way, just pass in to the `Config` constructor (followup will ensure that a `Config` passed into `new Config()` will end up unchanged).